### PR TITLE
Added missing registry key

### PR DESCRIPTION
**Pull request by dotMorten**
Added missing documentation regarding using the HKCU location rather than HKLM which will enables per-user installs to also use a registry location without requiring admin rights to run the installer.

/CC @unniravindranathan @emgarten

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/Visual-Studio-Extensibility/Visual-Studio-Templates.md](https://github.com/dotMorten/docs.microsoft.com-nuget/blob/patch-1/docs/Visual-Studio-Extensibility/Visual-Studio-Templates.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201705192152059424-344%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/Visual-Studio-Extensibility/Visual-Studio-Templates.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ SDKs that are installed using an MSI can install NuGet packages directly on the
 
 1. Write a registry key to identify the package location:
 
-    - Key location: `HKEY_LOCAL_MACHINE\SOFTWARE[\Wow6432Node]\NuGet\Repository`
+    - Key location: Either the machine-wide `HKEY_LOCAL_MACHINE\SOFTWARE[\Wow6432Node]\NuGet\Repository` or if it's per-user installed templates and packages, alternatively use `HKEY_CURRENT_USER\SOFTWARE\NuGet\Repository`
```
Comment (emgarten) [inline docs/Visual-Studio-Extensibility/Visual-Studio-Templates.md]: Is HKCU with Wow6432Node possible here?

**Comment by emgarten** [comment]
This update matches how NuGet reads the registry, for reference:
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio.Implementation/PreinstalledPackageInstaller.cs#L100-L106

File: docs/Visual-Studio-Extensibility/Visual-Studio-Templates.md (hunk 1 above)
Comment (dotMorten) [inline docs/Visual-Studio-Extensibility/Visual-Studio-Templates.md]: No didn't work. Also if setting this from the vsix it doesn't go in the wow node either. Don't think wow applies to the user part at all. It's more or less empty

Comment (dotMorten) [inline docs/Visual-Studio-Extensibility/Visual-Studio-Templates.md]: https://msdn.microsoft.com/en-us/library/windows/desktop/aa384253(v=vs.85).aspx

**Comment by kraigb** [comment]
@emgarten Can I assume you've reviewed and approved the change? If so, I'll merge.

**Comment by emgarten** [comment]
@kraigb yes, LGTM

**Pull request merged**